user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -305,279 +303,427 @@
                 <div style="font-weight:bold;">Time: 6:00 PM - 6:45 PM Saturday</div>
               </div>
               <div class="card-reveal">
-                <span class="card-title">Web Design Using CSS/HTML<i
+                <span class="card-title">3D Printing Overview<i
                   class="material-icons right">close</i></span>
-                <p>Have you ever wondered how to integrate CSS and HTML in creating
-                  your own website? Join us for the Web Design using CSS/HTML workshop,
-                  in which we teach all attendees how to create their own website!
-                  Little to no programming experience is required.
+                <p>3D printing or additive manufacturing is a new manufacturing technique
+                  used to produce more complex, unique parts. Along with that,
+                  it allows for hobbyist users to produce their own designs without
+                  an external firm to manufacture them, and increases the prevalence
+                  of rapid prototyping. Hackers can come and learn about 3D printing
+                  opportunities for computer science and engineering industries.
+                </p>
+                <p>
+                  There will be some free swag at the end so stop by!
                 </p>
-                <p>Uses: IDE of your choice (We will be using VSCode)</p>
+                <p class="card-requirements">This workshop has software requirements:</p>
+                <ul>
+                  <li class="wrksp-req">Fusion 360 account is necessary to participate in the designing portion.</li>
+                </ul>
               </div>
             </div>
             <div class="card beginner">
-              <div class="card-image waves-effect waves-block waves-light">
-                <img class="activator" src="../../../assets/images/workshop_leaders/RyanandBrie.png">
-                <a class="btn halfway-fab waves-effect waves-light red activator">click to learn more</a>
+              <div class="card-image">
+                <img class="activator" src="../../../assets/images/workshop_leaders/TiffanyKnearem.jpeg">
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
               </div>
               <div class="card-content pt-5">
-                <span class="card-title activator" style="font-weight:bold;">Creating in Adobe Illustrator</span>
-                <div style="font-weight:bold;">Ryan O'Neill and Bre Boeh</div>
-                <div style="font-weight:bold;">Location: 125 Business Building (Workshop Room 2)</div>
+                <span class="card-title activator" style="font-weight:bold;">Designing a Great User Experience</span>
+                <div style="font-weight:bold;">Tiffany Knearem, Google Cloud Platform Representative</div>
+                <div style="font-weight:bold;">Location: 124 Business Building (Workshop Room 2)</div>
```

Comment:
Room number and workshop room number are mismatched

Reply:
I really messed up that room number lmao. Fixed